### ISSUE TITLE
Hand made logo

### ISSUE THREAD

**Issue by 2bndy5** [cpp-linter/.github#4]
I have a Samsung Galaxy S10 with an sPen. And I wanted a large logo that could be scaled down (instead of scaled up). So, I drew this logo by hand on my phone:

![sketched_logo](https://user-images.githubusercontent.com/14963867/185709515-a0b99e05-3e59-4ea4-8a1f-9e7ce4628875.png)

This picture is a full resolution screenshot - I can export a psd file that has all the individual layers making up this image. The background can be easily removed and the old small logo is there for comparison.

I used the Adobe sketchbook app, and I hope there are no copyright limitations for content created with the app.

**Comment by 2bndy5** [comment]
This looks much nicer when it is scaled down...
As a logo in docs:
![image](https://user-images.githubusercontent.com/14963867/185711995-3f3d032a-0535-40d4-8a87-aa3786251478.png)
As a favicon:
![image](https://user-images.githubusercontent.com/14963867/185712086-1106c170-b224-4312-899e-c1fd60a99993.png)

**Comment by shenxianpeng** [comment]
You are a genius! Looks great. Only the full resolution screenshot, the edges of the circle, and the checkmark seem a little blurry.
Why don't I spend a dollar and let the designer on taobao.com deduct a full resolution logo for us to see the effect? let me try :)

**Comment by shenxianpeng** [comment]
This is a previous logo inspired by [sonarlint](https://images.g2crowd.com/uploads/product/image/social_landscape/social_landscape_e6066c64dded1b1af53469497010e534/sonarlint.png), I think changing "C/C++ lint" to "cpp linter" maybe the better.

![未标题-11_画板 1](https://user-images.githubusercontent.com/3353385/185820505-ca794b75-d682-44ae-a520-2b9f02938f83.png)

**Comment by 2bndy5** [comment]
> the edges of the circle, and the checkmark seem a little blurry.

There is A LOT of settings for each "brush" in the Adobe sketchbook app. I think there's a way to turn down the "blending" of stroke edges. After playing with it in GIMP, I was able to cut out the transparent "blur" around the strokes, but it doesn't look as organic as I had hoped.
![image](https://user-images.githubusercontent.com/14963867/185831985-6f01581b-f7a3-490e-a8b9-5c7bdc9dc3a1.png)

> Why don't I spend a dollar and let the designer on taobao.com create a full-resolution logo for us to see the effect? let me try :)

I'm not familiar with taobao.com (it looks like a chinese AliExpress after feeding it through google translator). Do they offer graphic design contracts? A dollar seems ridiculously cheap for such a service.

My main concern with using a image gotten from other sources was Licensing/Copyright infringement.

> logo inspired by [sonarlint](https://images.g2crowd.com/uploads/product/image/social_landscape/social_landscape_e6066c64dded1b1af53469497010e534/sonarlint.png), I think changing "C/C++ lint" to "cpp linter" maybe the better.

I like the squiggly line;  that's definitely achievable. I'll also make sure we're using a royalty-free font... TBC

**Comment by 2bndy5** [comment]
ok, so the squiggly line took me longer than I thought... This rough draft is using Century Gothic font (shipped with Windows I think), but I might go with something else.
![image](https://user-images.githubusercontent.com/14963867/185864120-b8b0c6fe-12b4-4b5b-8a88-6691f03a43e5.png)
![image](https://user-images.githubusercontent.com/14963867/185864199-14b22398-a1c3-4083-8bd3-7776a80b5f38.png)

**Comment by 2bndy5** [comment]
I started a branch here called "branding". And, I put the current WIP files (GIMP projects) in [the folder named "branding"](https://github.com/cpp-linter/.github/tree/branding/branding) (at repo root).

I highly suggest you install GIMP on your PC (its free and cross-platform compatible). Its very similar to Adobe Photoshop, so you shouldn't have too much trouble playing around or tweaking it.

**Comment by 2bndy5** [comment]
Just to showcase what you can do in GIMP, I made [this logo for the nRF24 org](https://github.com/nRF24/pyRF24/blob/main/docs/_static/Logo_large.jpg) using only a mouse/keyboard in GIMP.

**Comment by shenxianpeng** [comment]
I like the images updated later including no blurry logo, and the logo+cpp-linter+squiggly line.
It's almost perfect in my mind, probably the only thing that needs improvement is the font and text size, and spacing.

I have installed GIMP, but need some time to play with it.

> Do they offer graphic design contracts? A dollar seems ridiculously cheap for such a service.

I guess they can't offer a design contract, I'll check with them today, the dollar should be a waste, yours is good enough.

**Comment by shenxianpeng** [comment]
Create a new readme-banner
![readme-banner-2](https://user-images.githubusercontent.com/3353385/186060868-999f5464-c3c7-4fe0-927f-f975bb00cf5d.png)

**Comment by shenxianpeng** [comment]
Another one

![image](https://user-images.githubusercontent.com/3353385/186061808-8bbcd868-7e9c-47b6-b185-2e94df7453a1.png)

**Comment by 2bndy5** [comment]
My second choice for a font was Lato Bold:
![image](https://user-images.githubusercontent.com/14963867/186063234-815b8eb1-691d-44de-8760-040d449b745f.png)

**Comment by 2bndy5** [comment]
Did you want a more geometric squiggly line? If you push your changes to the GIMP file, I can do that for you.

**Comment by shenxianpeng** [comment]
The current font I sued is "Abril Fatface", I created the readme-banner with canva (here is the [link](https://www.canva.cn/design/DAFKGwyy65E/2Kb-L1BEWT2rCsdf2UxGlw/view?utm_content=DAFKGwyy65E&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton)). 

I'm still not familiar with how to use GIMP, so I picked a free squiggly from canva, it's okay for me to have a more geometric squiggly line.

**Comment by 2bndy5** [comment]
Thanks for the font name. It wasn't available in the system fonts, so I downloaded a copy and installed it to use the font in GIMP. We now have a large scale version that can be scaled down as needed (without ditortion):
![image](https://user-images.githubusercontent.com/14963867/186120569-4266d0ee-6efc-41a4-9e4a-3df68372a47e.png)
![image](https://user-images.githubusercontent.com/14963867/186120598-e5f2a38f-3e0b-4056-9f7e-85cd63c544a3.png)


I've saved a copy of the Abril Fatface font (with accompanied OTF License in the branding folder and pushed these changes to the branding branch. Going forward, I prefer you export images that have a transparent background. I've been posting these images here with a solid background so we can see how it looks in either light or dark themes.
